packages/ethereum-contracts/test/foundry/DeployerBase.t.sol

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: AGPLv3
```

Comment:
**Medium**: This abstraction might not be needed. It might be overkill and we could combine this with `FoundrySuperfluidTester`. This was created to keep some of the test contracts 'lighter'. Might not have been a worthy tradeoff though.